### PR TITLE
Tejas

### PR DESCRIPTION
**Pull request by Tejas-Choudekar**

### CODE REVIEW

File: product.1.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (arohablue) [inline product.1.html]: Please follow proper naming conventions for html files. i.e product related html files.

File: product.7.html [hunk 2]
```diff
@@ -6,10 +6,10 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		 <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
 
-		<title>Electro - HTML Ecommerce Template</title>
+		<title>Elite Computer</title>
 
  		<!-- Google font -->
- 		<link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700" rel="stylesheet">
+ 		<link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,800" rel="stylesheet">
```
Comment (arohablue) [inline product.7.html]: Why is this change required?

File: product.8.html [hunk 3]
```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
-		<meta charset="utf-8">
+		<meta charset="utf-9">
```
Comment (arohablue) [inline product.8.html]: utf-9 is not a valid charset. There are some unnecessary changes, due to the use of replaceall feature. This won't happen if you review the changes before committing them.

File: product.8.html [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		 <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
 
-		<title>Electro - HTML Ecommerce Template</title>
+		<title>Elite Computer</title>
 
  		<!-- Google font -->
- 		<link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700" rel="stylesheet">
+ 		<link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,900" rel="stylesheet">
```
Comment (arohablue) [inline product.8.html]: here

**Review by rhnmahajan83 — APPROVED** [review]
